animations-tutorial.md

```diff
@@ -0,0 +1,589 @@
+---
+layout: page
+title: Animations Tutorial
+permalink: /animations-tutorial/
+---
+
+Welcome to Flutter's Animations Tutorial. You may also be interested
+in our [Animations Overview](/animations/).
+
+## Introduction
+
+The animation system in Flutter is based on typed
+[`Animation`](http://docs.flutter.io/flutter/animation/Animation-class.html)
+objects that produce values over a certain duration. Widgets can either
+incorporate these animations in their `build`
+functions directly by reading their current value and listening to their
+state changes or they can use the animations as the basis of more elaborate
+animations that they pass along to other widgets.
+
+There are two separate animation libraries for Flutter. This documentation
+covers the [flutter/animation](http://docs.flutter.io/flutter/animation/animation-library.html) library, which is intended for animations in
+user interfaces that are built on StatefulComponent and StatelessComponent.
```

Comment:
Links to http://docs.flutter.io/flutter/widgets/StatefulComponent-class.html and http://docs.flutter.io/flutter/widgets/StatelessComponent-class.html ?